.github/workflows/tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
```

Comment:
Just out of interest: What does the `uses` flag do and why do we have two of them where one is with "-" and the other one without? 😄

Reply:
So every dash represents a step. The first step has a name and for the second I forgot to give a name. 
`uses` specifies an action that is used by the step. You can think of it like smaller workflows or code that does something. E.g.:
- `actions/checkout@v2` checks out the code in the git repo
- `actions/setup-node@v2` installs node on the `ubuntu-latest` machine - so that we can use node and npm in the next steps

@JenniferFalkenstein